packages/mobile/src/navigator/types.tsx

```diff
@@ -200,13 +201,17 @@ export type StackParamList = {
   [Screens.Send]:
     | {
         isOutgoingPaymentRequest?: boolean
+        skipContactsImport?: boolean
+        defaultCurrency?: Currency
```

Comment:
Is `defaultCurrency` being used? Otherwise lets remove it :)

Reply:
It was not being used. It's removed in https://github.com/valora-inc/wallet/pull/1036/commits/164434e67ea3dca9459b3cd20e1380a81d52c579.